From b2ae7a152921bf36a39f01de43769ee90cbbd253 Mon Sep 17 00:00:00 2001
From: Joseph Lansdowne <j49137@gmail.com>
Date: Tue, 9 Apr 2019 21:08:32 +0100
Subject: [PATCH] fix option `no_verify_hostname`

Broke with a curl upgrade at some point.  1 is no longer a valid option
- not sure exactly what it used to do.
---
 ChangeLog | 3 +++
 ftpfs.c   | 4 +---
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/ftpfs.c b/ftpfs.c
index 020e559..207d5fd 100644
--- a/ftpfs.c
+++ b/ftpfs.c
@@ -1627,9 +1627,7 @@ static void set_common_curl_stuff(CURL* easy) {
   }
 
   if (ftpfs.no_verify_hostname) {
-    /* The default is 2 which verifies even the host string. This sets to 1
-     * which means verify the host but not the string. */
-    curl_easy_setopt_or_die(easy, CURLOPT_SSL_VERIFYHOST, 1);
+    curl_easy_setopt_or_die(easy, CURLOPT_SSL_VERIFYHOST, 0);
   }
 
   curl_easy_setopt_or_die(easy, CURLOPT_INTERFACE, ftpfs.interface);
